modules/virt-overriding-default-fs-overhead-value.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-reserving-pvc-space-fs-overhead.adoc
+
+[id="virt-overriding-default-fs-overhead-value_{context}"]
+= Overriding the default file system overhead value
+
+Change the amount of persistent volume claim (PVC) space that the Containerized Data Importer (CDI) reserves for file system overhead by editing the `spec` attribute of the `CDI` object.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Open the `CDI` object for editing by running the following command:
++
+[source,terminal]
+----
+$ oc edit cdi
+----
+
+. Edit the `spec.config.filesystemOverhead` fields, populating them with your chosen values:
++
+[source,yaml]
+----
+...
+spec:
+  config:
+    filesystemOverhead:
+      global: "<new_global_value>" <1>
+      storageClass:
+        <storage_class_name>: "<new_value_for_this_storage_class>" <2>
+----
+<1> For example, `global: "0.07"` reserves 7% of the PVC for file system overhead.
+<2> For example, `mystorageclass: "0.04"` changes the default overhead value for PVCs in the `mystorageclass` storage class to 4%.
```

Comment:
It would be good to declare what these params are prior to these examples.
Something like:
<1> The file system overhead to be used globally across the cluster. For exam...
<2> The file system overhead for the specified storage class. This value overrides the global value. For exam...